client/src/state/campaign/CampaignActions.js

```diff
@@ -64,12 +66,22 @@ export function publishCampaign(publishApi, recordType, campaignId) {
           type: RECORD_ACTION_TYPES.FETCH_RECORD_SUCCESS,
           payload: { recordType, data },
         });
+        const message = i18n._t(
+          'CampaignAdmin.PUBLISH_SUCCESS',
+          '"%s" campaign has been published.'
```

Comment:
We need to ensure we are using the same patterns in the CMS, can we check what pages do? As this is not the same wording that was recommended by Jared.

Reply:
Translation is now `Published \"%s\" successfully.` which is the same message used when publishing a page.